playground/backend/internal/environment/environment_service.go

```diff
@@ -137,6 +140,17 @@ func GetNetworkEnvsFromOsEnvs() (*NetworkEnvs, error) {
 func ConfigureBeamEnvs(workDir string) (*BeamEnvs, error) {
 	sdk := pb.Sdk_SDK_UNSPECIFIED
 	preparedModDir, modDirExist := os.LookupEnv(preparedModDirKey)
+
+	countOfPossibleCodeProcessing := defaultCountOfPossibleCodeProcessing
```

Comment:
let's rename it to what Ilya proposed this morning - numOfParallelJobs

Reply:
Done.